PhotoBooth/CaptureSession.swift

```diff
@@ -0,0 +1,146 @@
+import UIKit
+import AVFoundation
+
+protocol PhotoCaptureable {
+    
+    func getPhotos() -> UIImage
+    
+    func configurePreview(view: UIView)
+}
+
+class CaptureSession: NSObject, PhotoCaptureable, AVCapturePhotoCaptureDelegate {
+    
+    var avSession = AVCaptureSession()
+    
+    var backCamera: AVCaptureDevice?
+    var frontCamera: AVCaptureDevice?
+    var currentCamera: AVCaptureDevice?
+    
+    var photoOutput: AVCapturePhotoOutput?
+    var videoOutput: AVCaptureVideoDataOutput?
+    let photoSessionPreset = AVCaptureSession.Preset.photo
+    let videoSessionPreset = AVCaptureSession.Preset.medium
+    
+    var cameraPreviewLayer: AVCaptureVideoPreviewLayer?
+    var cameraView = CameraView()
```

Comment:
I dont think the session should own the view.

Might be better to do VC <-> CameraView  <-> `CaptureSession: PhotoCapturable`
And make the camera view configurable with time and picture count

Reply:
can you explain this

Reply:
the Camera view could own the capture session and be responsible for the timed photos.
Or you could make the HomeViewController in to CameraViewController and have it own the capture session (PhotoCapturable) 

CameraViewController 
  -> CameraViewController.view 
  -> PhotoCapturable

It sets up the photoCapturable using it's own view.
It presents the form and uses the forms input to generate a configuration for the shoot.

This might be the simplest approach

Reply:
Yeah that might be the easiest for now